Selenium_Basics/EpamNavigationTests2.cs

```diff
@@ -0,0 +1,109 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Selenium_Basics;
+
+public class EpamNavigationTests2
+{
+    private IWebDriver driver { get; set; }
+
+    [SetUp]
+    public void SetupBrowser()
+    {
+        var options = new ChromeOptions();
+        options.BinaryLocation = "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome";
+        driver = new ChromeDriver(options);
+    }
+    
+    [TearDown]
+    public void TearDown()
+    {
+        driver.Quit();
+    }
+    
+    [Test]
+    public void CheckThatListOfCountriesContainsAmerEMEAandAPAC()
+    {
+        var epamUrl = "https://www.epam.com/";
+
+        driver.Manage().Window.Maximize();
+        driver.Navigate().GoToUrl(epamUrl);
+        driver.FindElement(By.XPath("//*[text()='Careers']//ancestor::*[@class='top-navigation__item-text']")).Click();
+        driver.FindElement(By.XPath("(//*[name()='use'])[10]")).Click();
+        bool Americas = driver.FindElement(By.XPath("//a[@data-item='0']")).Displayed;
+        bool EMEA = driver.FindElement(By.XPath("//a[@data-item='1']")).Displayed;
+        bool APAC = driver.FindElement(By.XPath("//a[@data-item='2']")).Displayed;
+        
+        Assert.True(Americas, "not displayed");
+        Assert.True(EMEA, "not displayed");
+        Assert.True(APAC, "not displayed");
+    }
+    
+    [Test]
+    public void CheckThatUrlToSearchContainsMyText()
+    {
+        var epamUrl = "https://www.epam.com/";
+        var searchUrl = "https://www.epam.com/search?q=Automation";
+        var keyWord = "Automation";
+            
+        driver.Manage().Window.Maximize();
+        driver.Navigate().GoToUrl(epamUrl);
+        driver.FindElement(By.XPath("//div[@class='header-search-ui header-search-ui-23 header__control']")).Click();
+        Thread.Sleep(1000);
+        driver.FindElement(By.XPath("//input[@id='new_form_search']")).Click();
+        driver.FindElement(By.XPath("//input[@id='new_form_search']")).SendKeys(keyWord);
+        driver.FindElement(By.XPath("//span[@class='bth-text-layer']")).Click();
+        
+        Assert.That(driver.Url,Is.EqualTo(searchUrl), "The url is wrong");
+
+        var ListOfArticles = driver.FindElements(By.XPath("//article")).ToList().Take(5);
+        var ArticlesToLower = ListOfArticles.Select(article=>article.Text.ToLower());
+        string Output = string.Join(",", ArticlesToLower);
+        foreach (var article in ArticlesToLower)
```

Comment:
why do you need this part of code? Is var a is used anywhere?
 foreach (var article in ArticlesToLower)
        {
            var a = article.Contains(keyWord.ToLower());
        }